sign/eddilithium3/eddilithium_test.go

```diff
@@ -0,0 +1,106 @@
+package eddilithium3
```

Comment:
Since there are no tests specific to the internals of the package.

```suggestion
package eddilithium3_test
```

Reply:
Sure.